### PR TITLE
Adding tools section

### PR DESCRIPTION
**Pull request by graceliching**
New tools section for easier navigation for users. Label changes to match the names of the tools.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -92,6 +92,84 @@
   * [List formulas](formulas/array-list-formulas.md)
   * [Other formulas](formulas/other-formulas.md)
   * [Complex data types](formulas/complex-data-types.md)
+* [Tools](tools.md)
+  * [Properties](features/account-properties.md)
```
Comment (huilinn) [inline SUMMARY.md]: Should this be `Account properties`?

File: SUMMARY.md [hunk 2]
```diff
@@ -92,6 +92,84 @@
...
+    * [Participants](workflow.md#using-people-task-as-an-approver)
+    * [Approver view](workflow.md#approver-view)
+    * [Approving tasks](workflow.md#approving-tasks)
+  * [Message templates](features/message-template.md)
+    * [Variables](features/message-template.md#variables)
+    * [Sections](features/message-template.md#sections)
+    * [Comments](features/message-template.md#comments)
+    * [Partials](features/message-template.md#partials)
+    * [Using templates in recipes](features/message-template.md#using-templates-in-recipes)
+  * [Schema Reuse](features/common-data-model.md)
```
Comment (huilinn) [inline SUMMARY.md]: "Schema reuse"

File: SUMMARY.md [hunk 3]
```diff
@@ -92,6 +92,84 @@
...
+  * [Properties](features/account-properties.md)
+  * [Lookup table](features/lookup-tables.md)
+    * [Setup](features/lookup-tables.md#setting-up-your-lookup-tables)
+    * [Add entry action](features/lookup-tables.md#add-entry-action)
+    * [Lookup entry action](features/lookup-tables.md#lookup-entry-action)
+    * [Search entry action](features/lookup-tables.md#search-entries-action)
+    * [Lookup table formulas](features/lookup-tables.md#lookup-table-formulas)
+    * [Table limits](features/lookup-tables.md#limits)
+  * [People Task](workflow.md)
+    * [Administrators](workflow.md#using-people-task-as-an-admin)
```
Comment (huilinn) [inline SUMMARY.md]: @eeshansim do you know if we can have 2 TOC links bringing us to the same page? Which TOC link on the nav bar would be highlighted in this case - the one the user clicked on?

File: SUMMARY.md (hunk 1 above)
Comment (graceliching) [reply SUMMARY.md]: Also, some of these guys have different names when you click on them in tools, eg. recipe lifecycle dev = import/export, properties = account properties, schema reuse = shared schema, on prem agent = on prem secure agent. I think we should change them to be consistent everywhere

**Pull request merged**